docs/ko/guides/glossary-guide.md

```diff
@@ -92,7 +92,7 @@
 **사전 순으로 용어집을 편집해주세요.**
 
 | 용어               | 번역            | 기타                                                                                                   |
-| ------------------ | --------------- | ------------------------------------------------------------------------------------------------------ | 
+| ------------------ | --------------- | ------------------------------------------------------------------------------------------------------ | --- |
```

Comment:
@queengooborg , @hochan222 

hello!

it breaks the markdown table layout.
i fixed it already but this code overwrite the changes

It seems to be fixed automatically, is there any modification I can do separately?

fix pr : https://github.com/mdn/translated-content/pull/14375

Reply:
Hmm, if Prettier keeps wanting to make this change, then there's some other problem with the table...  It looks like we have some extra columns on lines 118-119, which is what's causing this to constantly reappear.  I opened #14934 to apply a permanent fix!

Reply:
@queengooborg 

I checked the main branch for modifications!

thank you for your quick response. 👍